api/src/main/java/org/openmrs/validator/ObsValidator.java

```diff
@@ -66,6 +66,9 @@ public boolean supports(Class c) {
 	 */
 	public void validate(Object obj, Errors errors) {
 		Obs obs = (Obs) obj;
+		if(obs.getVoided()){
+			return;
+		}
 		List<Obs> ancestors = new ArrayList<Obs>();
 		//ancestors.add(obs);
 		validateHelper(obs, errors, ancestors, true);
```

Comment:
You need to update `validateHelper()` to call validate recursively for the child obs so that they also get skipped if voided